ui/src/app/shared/pipes/cesure.pipe.ts

```diff
@@ -0,0 +1,32 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({
+  name: 'cesureLeft'
+})
+export class CesureLeftPipe implements PipeTransform {
+
+  transform(value: number | string, size: number = 40): string {
+    value = value.toString();
+    if (value.length <= size) {
+      return value;
+    }
+    return '... ' + value.substr(value.length - size, size);
```

Comment:
I think this is fine for now BUT - Eventually we should be mindful of multi-byte characters. Definitely a follow-up issue - Maybe consider using something like https://github.com/bestiejs/punycode.js. This is a general issue across the app.